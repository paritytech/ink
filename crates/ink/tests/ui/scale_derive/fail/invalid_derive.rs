#[ink::scale_derive(Encode, Whatevs)]
struct S;

fn main() {
}