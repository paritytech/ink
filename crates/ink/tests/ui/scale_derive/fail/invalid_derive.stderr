error: unsupported scale derive: expected Encode, Decode or TypeInfo
 --> tests/ui/scale_derive/fail/invalid_derive.rs:1:29
  |
1 | #[ink::scale_derive(Encode, Whatevs)]
  |                             ^^^^^^^
